static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -88,13 +105,17 @@ function SentryFunctionDetails(props: Props) {
     res.status(200).send(message);
   };`;
 
-  const [events, setEvents] = useState(sentryFunction?.events || []);
+  form.current.getEnvVariables = () => {
```

Comment:
@vpandiarajan20 this feels a bit hacky to add these functions to the form. Is there no other strategy we can use?